src/rust/ensogl/src/display/render/pipeline.rs

```diff
@@ -56,16 +56,18 @@ pub struct RenderPassOutput {
     pub internal_format : AnyInternalFormat,
     /// Item texture type of the pass framebuffer's attachment.
     pub item_type : AnyItemType,
+    /// Texture parameters that will be set when binding the texture.
+    pub parameters: Parameters,
```

Comment:
What are "pipeline parameters"? See later comments - they explain the issue better.